arviz/plots/khatplot.py

```diff
@@ -35,60 +35,57 @@ def plot_khat(
     show=None,
     **kwargs
 ):
-    r"""
-    Plot Pareto tail indices for diagnosing convergence.
+    r"""Plot Pareto tail indices :math:`\hat{k}` for diagnosing convergence in PISIS-LOO.
 
     Parameters
     ----------
-    khats : ELPDData containing Pareto shapes information or array of
-        Pareto tail indices.
-    color : str or array_like, optional
-        Colors of the scatter plot, if color is a str all dots will
-        have the same color, if it is the size of the observations,
-        each dot will have the specified color, otherwise, it will be
-        interpreted as a list of the dims to be used for the color
-        code. If Matplotlib c argument is passed, it will override
-        the color argument
-    xlabels : bool, optional
-        Use coords as xticklabels
-    show_hlines : bool, optional
+    khats : ELPDData containing Pareto shapes information or array of Pareto tail indices.
```

Comment:
```suggestion
    khats : ELPDData or array-like
        The input Pareto tail indices to be plotted. It can be an ``ELPDData`` object containing
        Pareto shapes or an array. In this second case, all the values in the array are interpreted
        as Pareto tail indices.
```